File: frontend/src/components/items/illusions/time_stop.js
export const timeStop = {
  key: 'time_stop',
  name: 'Time Stop',
  category: 'illusion',
  description: 'Time slows, delaying result reveals.',
};

export const hasTimeStop = (targetEffects) =>
  targetEffects.some((entry) => entry.item_key === 'time_stop');

export const TIME_STOP_REVEAL_DELAY_MS = 1200;

File: backend/app/items/illusions/time_stop.py
def _time_stop(conn, user_id: int, campaign_id: int):
    return {}


time_stop_item = {
    "key": "time_stop",
    "name": "Time Stop",
    "description": "Slow down letter reveal timing for each guess.",
    "cost": 2,
    "category": "illusion",
    "handler": _time_stop,
    "affects_others": True,
    "requires_target": True
}
